src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/OrderStates/OrderStateType.php

```diff
@@ -0,0 +1,214 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Configure\ShopParameters\OrderStates;
+
+use PrestaShop\PrestaShop\Adapter\Configuration;
+use PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints\DefaultLanguage;
+use PrestaShop\PrestaShop\Core\ConstraintValidator\Constraints\TypedRegex;
+use PrestaShop\PrestaShop\Core\MailTemplate\Layout\Layout;
+use PrestaShop\PrestaShop\Core\MailTemplate\ThemeCatalogInterface;
+use PrestaShopBundle\Form\Admin\Type\ColorPickerType;
+use PrestaShopBundle\Form\Admin\Type\TranslatableChoiceType;
+use PrestaShopBundle\Form\Admin\Type\TranslatableType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Routing\Router;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Type is used to created form for order state add/edit actions
+ */
+class OrderStateType extends TranslatorAwareType
+{
+    /**
+     * @var array
+     */
+    private $templates;
+
+    /**
+     * @var Router
+     */
+    private $routing;
+
+    /**
+     * @var array
+     */
+    private $templateAttributes;
+
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        ThemeCatalogInterface $themeCatalog,
+        Router $routing
+    ) {
+        parent::__construct($translator, $locales);
+        $this->routing = $routing;
+        $mailTheme = Configuration::get('PS_MAIL_THEME', 'modern');
+
+        $mailLayouts = $themeCatalog->getByName($mailTheme)->getLayouts();
+
+        foreach ($locales as $locale) {
+            $languageId = $locale['id_lang'];
+            $this->templates[$languageId] = [];
+            $this->templateAttributes[$languageId] = [];
```

Comment:
```suggestion
            $this->templates[$languageId] = $this->templateAttributes[$languageId] = [];
```

Reply:
I personally find this single line harder to read than the 2 aboves 🤔